src/modules/flight_mode_manager/tasks/Descend/FlightTaskDescend.cpp

```diff
@@ -36,6 +36,13 @@
 
 #include "FlightTaskDescend.hpp"
 
+bool FlightTaskDescend::activate(const trajectory_setpoint_s &last_setpoint)
```

Comment:
Had to rebase to resolve conflict that came in with https://github.com/PX4/PX4-Autopilot/pull/21192. There the activate() was removed, so I add it back here.